### PR TITLE
Fix polkadot e2e tests

### LINKED ISSUE #4778: Polkadot e2e test are failing
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: 
 - Browser: 
 - Version: 

**Smartphone (please complete the following information):**
 - Device: 
 - OS: 
 - Browser: 
 - Version: 

**Additional context**

### PR DESCRIPTION
**Pull request by mariopino**
Closes #4778 

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: app/tests/e2e/helpers.js [hunk 1]
```diff
@@ -132,6 +132,7 @@ async function actionModalCheckout(
     ".action-modal-footer .button:nth-of-type(2):enabled",
     30000
   )
+  await browser.pause(500)
```
Comment (faboweb) [inline app/tests/e2e/helpers.js:135]: what are we waiting for here?

**Comment by faboweb** [comment]
🚀 Deployed on https://5f493f7b004db080bb87dd67--lunieio.netlify.app

File: app/tests/e2e/helpers.js (hunk 1 above)
Comment (mariopino) [reply app/tests/e2e/helpers.js:135]: Test stops at this point, I'm still testing but seems some query is not loade. Then when click on the Next button it doesn't do anything and therefore you don't get the invoice.

Comment (faboweb) [reply app/tests/e2e/helpers.js:135]: Can we wait explicit for the balances to have been loaded?

Comment (mariopino) [reply app/tests/e2e/helpers.js]: Great @faboweb , that pause was not the best way to fix the problem. Disabling next button until balances loaded in action modal is a cleaner way so far.

**Pull request merged**